### DISCUSSION TITLE
Can you provide different color variants of the superset logo?

### DISCUSSION THREAD

**Discussion by GentleGhostCoder** [apache/superset#20956] [Ideas]
It would be great if there were different color variants of the Superset logo (e.g. for dark mode etc.).
Currently I'm afraid that doing this (and publishing it) would be a violation of the Apache Software Foundation Trademark Policy.
Personally, I would like the main color (lettering and outer component - currently in #484848) to be in white.

Thanks in advance :)

**Comment by florieger** [comment]
Might be related to https://github.com/apache/superset/discussions/19175
